packages/bip32/src/index.ts

```diff
@@ -54,7 +54,7 @@ export const onRpcRequest: OnRpcRequestHandler = async ({ request }) => {
         params: [
           {
             prompt: 'Signature request',
-            textAreaContent: `Do you want to ed25519 sign ${message} with ${add0x(
+            textAreaContent: `Do you want to ${curve} sign ${message} with ${add0x(
```

Comment:
It's always signed with `ed25519` 🤔

Reply:
I added a button to sign with `secp256k1`, but I guess that doesn't make much sense if we still use the `ed25519` package to sign 🤔

Reply:
That's what I mean yeah